### PR TITLE
chore(llc): Regenerated open-api models

### PR DESCRIPTION
**Pull request by Brazol**

### CODE REVIEW

File: packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart [hunk 1]
```diff
@@ -272,14 +278,17 @@ class OwnUserResponse {
       // Note 2: this code is stripped in release mode!
       assert(() {
         requiredKeys.forEach((key) {
-          //assert(json.containsKey(key), 'Required key "OwnUserResponse[$key]" is missing from JSON.');
-          //assert(json[key] != null, 'Required key "OwnUserResponse[$key]" has a null value in JSON.');
+          assert(json.containsKey(key), 'Required key "OwnUserResponse[$key]" is missing from JSON.');
+          assert(json[key] != null, 'Required key "OwnUserResponse[$key]" has a null value in JSON.');
```
Comment (deven98) [inline packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart:282]: Have you verified these keys now exist in the response?

File: packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart [hunk 2]
```diff
@@ -288,25 +297,25 @@ class OwnUserResponse {
...
         role: mapValueOfType<String>(json, r'role')!,
         teams: json[r'teams'] is Iterable
             ? (json[r'teams'] as Iterable).cast<String>().toList(growable: false)
             : const [],
-        totalUnreadCount: mapValueOfType<int>(json, r'total_unread_count') ?? 0,
-        unreadChannels: mapValueOfType<int>(json, r'unread_channels') ?? 0,
-        unreadThreads: mapValueOfType<int>(json, r'unread_threads') ?? 0,
+        totalUnreadCount: mapValueOfType<int>(json, r'total_unread_count')!,
+        unreadChannels: mapValueOfType<int>(json, r'unread_channels')!,
+        unreadThreads: mapValueOfType<int>(json, r'unread_threads')!,
```
Comment (deven98) [inline packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart:318]: Please make sure all these values exist before scrapping the defaults

File: packages/stream_video/lib/open_api/video/coordinator/model/user_updated_event.dart [hunk 3]
```diff
@@ -78,8 +84,9 @@ class UserUpdatedEvent {
 
       return UserUpdatedEvent(
         createdAt: mapDateTime(json, r'created_at', r'')!,
+        receivedAt: mapDateTime(json, r'received_at', r''),
         type: mapValueOfType<String>(json, r'type')!,
-        user: UserObject.fromJson(json[r'user']),
+        user: UserEventPayload.fromJson(json[r'user'])!,
```
Comment (deven98) [inline packages/stream_video/lib/open_api/video/coordinator/model/user_updated_event.dart:89]: Please make sure the user payload is now returned from the user updated event

**Comment by deven98** [comment]
Additionally, can you just do one final check that everything in the WS event works well?

File: packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart (hunk 1 above)
Comment (Brazol) [reply packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart:282]: Tested those on dogfooding, works fine

File: packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart (hunk 2 above)
Comment (Brazol) [reply packages/stream_video/lib/open_api/video/coordinator/model/own_user_response.dart:318]: Tested those on dogfooding, works fine

**Review by deven98 — APPROVED** [review]
LGTM

**Pull request merged**